### PR TITLE
fix(GKE): Cert data was stored as []byte but needed to be a string

### PR DESCRIPTION
**Pull request by jchauncey**
The kubeconfig structs stored the client auth data as []bytes which caused issues when trying to marshal to yaml.

### CODE REVIEW

**Comment by jchauncey** [comment]
@arschles check this out and let me know what you think.

**Comment by arschles** [comment]
@jchauncey :+1: on creating custom types for generating the kubeconfig file. I had considered that but the ones in the `k8s.io` package worked at the time. I'm good with this

File: handlers/gke.go [hunk 1]
```diff
@@ -20,6 +21,85 @@ var (
 	errUnusedGKEClusterNotFound = errors.New("all GKE clusters are in use")
 )
 
+// Config holds the information needed to build connect to remote kubernetes clusters as a given user
```
Comment (vdice) [inline handlers/gke.go]: looks like an errant `build` in this comment?

Comment (jchauncey) [inline handlers/gke.go]: k fixed (copy pasta from k8s lib)

**Comment by arschles** [comment]
@jchauncey when you merge this, can you clean up commit messages (or squash and provide a cleaner message) that conform to the standard?

**Pull request merged**